packages/mobile/src/components/AccountNumber.tsx

```diff
@@ -2,31 +2,58 @@ import colors from '@celo/react-components/styles/colors.v2'
 import fontStyles from '@celo/react-components/styles/fonts.v2'
 import { getAddressChunks } from '@celo/utils/src/address'
 import React from 'react'
-import { StyleSheet, Text, View } from 'react-native'
+import { useTranslation } from 'react-i18next'
```

Comment:
Nothing to do with your PR but just noting that `src/components` gives us no information 😄

Reply:
Sorry I don't follow

Reply:
haha i just want to get rid of our `src/components` folder because it is not a helpful name - basically everything is a component